mysql-test/std_data/dd/sdi/innodb_sdi/mysql.json

```diff
@@ -15,7 +15,7 @@
         "created": NNN,
         "last_altered": NNN,
         "hidden": 2,
-        "options": "avg_row_length=0;explicit_tablespace=1;key_block_size=0;keys_disabled=0;pack_record=1;row_type=2;stats_auto_recalc=0;stats_persistent=0;stats_sample_pages=0;",
+        "options": "avg_row_length=0;encrypt_type=N;explicit_tablespace=1;key_block_size=0;keys_disabled=0;pack_record=1;row_type=2;stats_auto_recalc=0;stats_persistent=0;stats_sample_pages=0;",
```

Comment:
why do we have this DD change? AFAIU, upstream adds encrypt_type=N for all non-encrypted tables.

Reply:
Upstream did - but we not have fully merged upstream changes before this PR. This comes from upstream and it was merged in from upstream in this PR.